cachito/web/api_v1.py

```diff
@@ -210,7 +211,16 @@ def create_request():
         ]
     )
 
-    chain(chain_tasks).delay()
+    try:
+        chain(chain_tasks).delay()
+    except kombu.exceptions.OperationalError:
+        flask.current_app.logger.exception(
+            "Failed to schedule the task for request %d. Failing the request.", request.id
+        )
+        error = "Failed to schedule the task to the workers. Please try again."
```

Comment:
Could Cachito try again?

Reply:
Celery will automatically retry for us.
https://github.com/release-engineering/cachito/blob/e1d7846bf7e1a896baa338fbb1b45880bb0c5150/cachito/workers/config.py#L18